test/urdf/Acrobot_with_limits.urdf

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+
+<robot name="DoublePendulum">
+  <link name="base_link">
+    <visual>
+      <geometry>
+         <box size=".2 .2 .2" />
+      </geometry>
+      <material name="green">
+	       <color rgba="0 1 0 1" />
+      </material>
+    </visual>
+  </link>
+
+  <link name="upper_link">
+    <inertial>
+      <origin xyz="0 0 -.5" rpy="0 0 0" />
+      <mass value="1" />
+      <inertia ixx="1" ixy="0" ixz="0" iyy="0.083" iyz="0" izz="1" />
+    </inertial>
+    <visual>
+      <origin xyz="0 0 -.5" rpy="0 0 0" />
+      <geometry>
+         <cylinder length="1.1" radius=".05" />
+      </geometry>
+      <material name="red">
+	       <color rgba="1 0 0 1" />
+      </material>
+    </visual>
+  </link>
+
+  <link name="lower_link">
+    <inertial>
+      <origin xyz="0 0 -1" rpy="0 0 0" />
+      <mass value="1" />
+      <inertia ixx="1" ixy="0" ixz="0" iyy="0.33" iyz="0" izz="1" />
+    </inertial>
+    <visual>
+      <origin xyz="0 0 -1" rpy="0 0 0" />
+      <geometry>
+         <cylinder length="2.1" radius=".05" />
+      </geometry>
+      <material name="blue">
+	       <color rgba="0 0 1 1" />
+      </material>
+    </visual>
+  </link>
+
+  <joint name="shoulder" type="continuous">
+    <parent link="base_link" />
+    <child link="upper_link" />
+    <origin xyz="0 .15 0" />
+    <axis xyz="0 1 0" />
+    <limit lower="-6.28" upper="6.28" velocity="10" effort="0"/>
+    <dynamics damping="0.1" />
+  </joint>
+
+  <joint name="elbow" type="continuous">
+    <parent link="upper_link" />
+    <child link="lower_link" />
+    <origin xyz="0 .1 -1" />
+    <axis xyz="0 1 0" />
+    <limit lower="-6.28" upper="6.28" velocity="10" effort="5"/>
```

Comment:
These velocity and effort limits are totally arbitrary. I haven't tested whether swing-up or balance are possible with these particular values.